From 0aa4efe1deb2fe6bfcb51be19e876f85738078e2 Mon Sep 17 00:00:00 2001
From: Mehdi Sabwat <mehdisabwat@gmail.com>
Date: Thu, 3 Oct 2019 22:32:12 +0200
Subject: [PATCH 4/6] wip: delete call to msg_Dbg

The logger module needs to be fixed, there is something wrong with it.
some calls to msg_Dbg on various places
(audio output/audio filters/MP4 demux/access, keystore, and maybe more...)
triggers Runtime Errors (out of bounds).
Also, we see garbage values in the console instead of actual output.
---
 modules/audio_output/openal.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/modules/audio_output/openal.c b/modules/audio_output/openal.c
index c48c5a0f5e..2a5f6f946d 100644
--- a/modules/audio_output/openal.c
+++ b/modules/audio_output/openal.c
@@ -191,7 +191,8 @@ static int Start( audio_output_t *aout, audio_sample_format_t *restrict fmt )
 	alSource3f(sys->source, AL_VELOCITY, 0, 0, 0);
 
 	alcGetIntegerv(sys->dev, ALC_FREQUENCY, 1, &sys->freq);
-	msg_Dbg(aout, "Openal Dev freq: %d", sys->freq);
+	
+	//msg_Dbg(aout, "Openal Dev freq: %d", sys->freq);
 
 	sys->start_date = 0;
 	sys->next_buffer = 0;
@@ -298,7 +299,7 @@ static int Time_Get( audio_output_t *aout, mtime_t *delay)
  *****************************************************************************/
 static void Close( vlc_object_t *obj )
 {
-	msg_Dbg(obj, "Openal Close");
+  //msg_Dbg(obj, "Openal Close");
 	audio_output_t *aout = (audio_output_t *)obj;
 	struct aout_sys_t *sys = aout->sys;
 
-- 
2.23.0

